roles/core/dhcp_server/molecule/default/molecule.yml

```diff
@@ -0,0 +1,100 @@
+dependency:
+  name: galaxy
+driver:
+  name: docker
+platforms:
+  - name: management1
+    groups:
+      - mg_managements
+    image: "quay.io/actatux/ansible-${MOLECULE_DISTRO:-centos:8}"
+    override_command: false
+    capabilities:
+      - "SYS_ADMIN"
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true
+
+    docker_networks:
+      - name: bbnet
+        ipam_config:
+          - subnet: '10.10.0.0/16'
+            gateway: "10.10.0.254"
+    networks:
+      - name: bbnet
+        ipv4_address: "10.10.0.1"            
+
+  - name: c001
```

Comment:
All prepare/converge/verifiy playbooks run on management1. What is the purpose of these additional containers?
If you need extra fake hosts, no need to spawn containers, you can add some to your inventory. (see [ssh_master](https://github.com/bluebanquise/bluebanquise/blob/0d0d2a08f3acea5a45215b4a70ad0268392061ba/roles/core/ssh_master/molecule/default/molecule.yml#L74))

Reply:
I am injecting all inventories from `resources/examples/simple_cluster/inventory/cluster/nodes`. For now these containers are only reflecting the simple cluster architecture from the example.

Reply:
I don't understand the purpose of these containers in the test. Why would we need to spawn these containers to test the role `dhcp_server`?

Reply:
I wanted to add some more advanced tests from clients to management, that is why I added extra containers.
Anyway, for now I believe that the number of tests that are being run in management are quite enough.
We will run more advanced functional tests in our own platform with kitchen.

Client containers removed in commit fd4d355

Reply:
Great. I'm not against adding client/server tests, while it can be challenging with containers. If so, add the containers when they are effectively requested for the test. Thanks. :wink: